include/Decimal.hpp

```diff
@@ -198,6 +202,15 @@ class Decimal {
         min.SetMin();
         return getDecimal() == min;
     }
+
+    bool operator==(const Decimal &other) {
+        return ::memcmp((void *) m_data, other.m_data, sizeof(m_data)) == 0;
```

Comment:
I will look at that. It uses ttmath internally to represent as we do in EE. though EE uses ttmath INT == operator for comparison vs the approach taken here.